### DISCUSSION TITLE
problème payement refusé sur paygreen  intégration sur woocommerce

### DISCUSSION THREAD

**Discussion by platup28** [PayGreen/documentation#40] [Q&A]
Bonjour,

Les paiements sont refusés, et les clients n’arrivent pas à payer. Pourtant, les clés API sont bien configurées sur WooCommerce. J’ai contacté le service client, mais je n’ai toujours pas reçu de réponse.

Merci d'avance pour votre aide !

**Comment by RomainPayGreen** [comment]
Bonjour,

Votre client a un “Solde insuffisant” sur sa carte donc le paiement a été refusé.

Bonne journée

**Reply by platup28** [reply]
Bonjour,

Les clients me disent le contraire. Pourrait-on effectuer un test en conditions réelles ? Mon compte est-il bien activé ?

Je perds des clients, et cela vous impacte également. Je vous serais reconnaissant de me prêter davantage d’attention à ce sujet.

Merci.

**Reply by RomainPayGreen** [reply]
Bonjour,

Je ne vois que deux clients différents avec tous les deux un soucis de solde, c'est assez peu pour dire qu'il y a un problème généralisé. Essayez de faire une tentative de votre côté avec une carte fonctionnelle dont vous êtes sûr du solde et je vérifierai les logs s'il y a encore des erreurs.
Tout est correctement activé dans votre boutique.

Bonne journée
